### DISCUSSION TITLE
Issues with the 86box

### DISCUSSION THREAD

**Discussion by crlcolunga** [86Box/86Box#2679] [General]
It's about the compaq presario emulated when I selected this machine to put the speed to 300, it's so very slow the virtual machine. I refere to socket 7 (dual voltage) (single voltage) supersocket 7 and other machine categories.

![image](https://user-images.githubusercontent.com/94066997/190831722-fd09defa-12b3-423d-8cc7-ea42d4708dfa.png)

check my spects.

What recomends?

**Comment by OBattler** [comment]
That has a much lower single thread performance than even my Nehalem Xeons, and they can run a Pentium 75 at most. At this point, your best best is a low-speed 486.
